thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/tasks/FunctionCreationOnboardingTask.java

```diff
@@ -238,24 +249,28 @@ protected AnomalyFunctionDTO getDefaultFunctionSpecByTimeGranularity(TimeGranula
         anomalyFunctionSpec.setCron("0 0 0 * * ?");
         anomalyFunctionSpec.setWindowSize(6);
         anomalyFunctionSpec.setWindowUnit(TimeUnit.HOURS);
+        anomalyFunctionSpec.setProperties("signTestWindowSize=24;anomalyRemovalThreshold=0.6;baseline=w/3wAvg;decayRate=0.5;signTestStepSize=1;slidingWindowWidth=8;confidenceLevel=0.99");
```

Comment:
Hi Yung-Yu, after a second think, these properties should have their own default values when using function. e.g. anomalyRemovalThreshold has defined in the class "AnomalyRemovalByWeight" in AD library. I recall that you have fully tested to trigger these anomaly function without any properties and they can still properly run. Here can we utilize the default values in AD library without writing explicitly these property values?

Reply:
Yes. you are right. I also checked my code, these configurations are inserted in the init(). We should look into what happened to the exceptions during replay.

Reply:
I just found out the code is still lying there for review.

Reply:
Thanks Yung-Yu. Since we are having the default setup in AD library, how about within this pr only merge the wrapper code for deploy? Thanks!

Reply:
Thanks Kexin for the comment. The rb has been pushed. I removed the default properties here.